src/Sticky/README.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 'Sticky'
+type: 'component'
+components:
+- Sticky
+categories:
+- Content
+status: 'New'
+designStatus: 'Done'
+devStatus: 'In progress'
+notes: |
+---
+
+Makes elements remain at the top or bottom of the viewport, like a sticky navbar.
+
+### Basic Usage
+
+### Top positioning
```

Comment:
These secondary headings should be one level below the primary "Basic Usage" heading level.